multi-transfer-esdt/src/lib.rs

```diff
@@ -87,14 +95,17 @@ pub trait MultiTransferEsdt:
             }
 
             // emit event before the actual transfer so we don't have to save the tx_nonces as well
-            self.transfer_performed_event(
-                batch_id,
-                eth_tx.from.clone(),
-                eth_tx.to.clone(),
-                eth_tx.token_id.clone(),
-                eth_tx.amount.clone(),
-                eth_tx.tx_nonce,
-            );
+            // emit events only for non-SC destinations
+            if self.blockchain().is_smart_contract(&eth_tx.to) {
```

Comment:
This solves issue 9 from Audit [document](https://docs.google.com/document/d/1-KDYd9DR9vKQnw08mzifbeVJ9TAIiOrvvX3MvWmEjsA/edit?tab=t.0#heading=h.3cq62w697v91).

Reply:
this is still wrong and backwards incompatible with other integrations like bridge-api or dune analytics

Reply:
To solve what @dragos-rebegea mentioned and fix the audit finding #9, do you think it would be better to do the following:
* emit the event only if the destination address is not a SC
* remove the `transfer_performed_sc_event` and instead emit the `transfer_performed_event` in the bridge proxy if the call to the SC succeeded

Reply:
Added events to Bridge Proxy SC, as discussed.